lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -86,7 +86,8 @@ def call(params)
             ignore_missing_values: params.ignore_missing_values == 'true' ? true : false,
             do_not_touch_filters_that_are_not_mentioned: params.do_not_touch_filters_that_are_not_mentioned == 'true' ? true : false,
             domain: domain,
-            dry_run: false
+            dry_run: false,
+            users_brick_input: params.users_brick_users
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)